.github/pull_request_template.md

```diff
@@ -3,6 +3,7 @@
 - [ ] **Quality**: This PR builds and tests run cleanly
   - `cargo clean; cargo test --all` runs without emitting any warnings
   - `cargo fmt` does not produce any changes to the code
+  - `./gradlew ktlint detekt` runs without emitting any warnings
```

Comment:
OH, that's where this is. This is super unrelated but can you add a step for `cargo clippy --all --all-targets --all-features`